diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..e316f38 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import arduinoScratchInsetIconURL from './arduino_scratch/arduino_scratch_icon.png';
+import arduinoScratchIconURL from './arduino_scratch/arduino_scratch_background.png';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -47,6 +50,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: 'Arduino Scratch',
+        extensionId: 'arduinoScratch',
+        iconURL: arduinoScratchIconURL,
+        insetIconURL: arduinoScratchInsetIconURL,
+        description: 'Connects to Arduinos running Firmata',
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
